mesh/ncmesh.hpp

```diff
@@ -21,6 +21,7 @@
 #include "vertex.hpp"
 #include "../fem/geom.hpp"
 
+#include <cstdint>
```

Comment:
This is not needed because of the transitive include from `hash.hpp`.

Reply:
I guess by https://google.github.io/styleguide/cppguide.html#Include_What_You_Use I'd still encourage having it.

Reply:
Yeah, makes sense. Reverted these in [2e77bdd](https://github.com/mfem/mfem/pull/3903/commits/2e77bdde72593ea9d78396294a33edc4c7f2bf48).